.github/workflows/build-dev-image.yml

```diff
@@ -46,15 +46,20 @@ jobs:
           export PATH=.:$PATH:phantomjs/bin
           jcli cwp --install-artifacts --config-path formula.yaml \
             --value-set version=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
+
+          # there's no a release for the multi-arch building but https://github.com/jenkinsci/custom-war-packager/releases/tag/v2.0-alpha-6
+          wget https://github.com/LinuxSuRen/custom-war-packager/releases/download/v2.0-alpha-6/custom-war-packager-cli-2.0-alpha-7-20210527.031403-1-jar-with-dependencies.jar
+          mv custom-war-packager-cli-2.0-alpha-7-20210527.031403-1-jar-with-dependencies.jar ~/.jenkins-cli/cwp-cli.jar
           jcli cwp --install-artifacts --config-path formula-arm.yaml --value-set output=push \
             --value-set version=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
       - name: Push Image
         run: |
           version=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
```

Comment:
I suggest that you get image version in this way:

```js
 version=${{ fromJSON(steps.meta.outputs.version }}
```

See also [here](https://github.com/docker/metadata-action#outputs).

Reply:
Thanks for your suggestion. Great!